### PR TITLE
Remote config crash when retrieving integer value

### PR DESCRIPTION
**Pull request by pantstamp**
Testing PR for this FluxC [PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/3017).

Check the other PR for context. 

Testing Steps: 
1. Install the app from this PR.
2. Login with an account.
3. Check that after login the app does not crash and you can see this line in the logcat. 
 `RemoteConfigTest        com.jetpack.android.prealpha         D  getPhaseFourOverlayFrequencyConfig value = 7`

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">This PR is tagged with <code>Do Not Merge</code> label(s).</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is not assigned to a milestone.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by zwarm — APPROVED** [review]
Works as expected - no more floats!!

:) Friendly reminder to take out the "Log" line in WPMainActivity before merging

**Comment by pantstamp** [comment]
@zwarm thanks for reviewing the PR and for testing! This PR is not for merging. All changes were needed just for us to be able to test if calling an integer remote config value works without crashing the app.

**Comment by pantstamp** [comment]
Closing this PR as it was only for testing.

**Pull request closed without merging**
